testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/assertions/ProcessInstanceAssertj.java

```diff
@@ -266,7 +266,7 @@ private static boolean isEnded(final ProcessInstanceDto processInstance) {
   private static String formatState(final ProcessInstanceState state) {
     if (state == null || state == ProcessInstanceState.UNKNOWN_ENUM_VALUE) {
       return "not created";
-    } else if (state == ProcessInstanceState.CANCELED) {
+    } else if (state == ProcessInstanceState.TERMINATED) {
       return "terminated";
     } else {
       return state.name().toLowerCase();
```

Comment:
🙃 This can be simplified now, I think
```suggestion
    } else {
      return state.name().toLowerCase();
```